eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/http/RequestUriAndTimeLogging.java

```diff
@@ -28,22 +29,24 @@
 
 import java.io.IOException;
 import java.net.URI;
+import java.util.concurrent.atomic.AtomicInteger;
 
 
 /**
- * {@link HttpRequestExecutor} decorator that logs request uris to logcat.
+ * {@link HttpRequestExecutor} decorator that logs request uris and round trip times to logcat.
  *
  * @author Gabor Keszthelyi
  */
-public final class RequestUriLogging implements HttpRequestExecutor
+public final class RequestUriAndTimeLogging implements HttpRequestExecutor
 {
+    private static AtomicInteger sRequestCounter = new AtomicInteger(1000);
```

Comment:
I am not sure why I did that but probably because somehow that looks more readable for me when it starts from a higher decimal, so you don't have 1 and 11 for example. I didn't use 100 to avoid confusion with http codes.
But it doesn't matter of course, so I change it to 1 now, also the back arrow is at the end, that looks better with the one digit numbers:

```
05-31 12:09:49.268 D/EventDiscovery-Request: (1->) POST https://api.schedjoules.com/insights?u=0e61014b-bf7c-4e9d-b2be-9b4a057533b6
05-31 12:09:49.696 D/EventDiscovery-Request: (1<-) Response time: 427 ms
05-31 12:09:51.124 D/EventDiscovery-Request: (2->) GET https://api.schedjoules.com/events?latlng=52.370216,4.895168&start_at_or_after=2017-05-31T10:09:50Z&radius=10000&u=0e61014b-bf7c-4e9d-b2be-9b4a057533b6
05-31 12:09:51.797 D/EventDiscovery-Request: (2<-) Response time: 673 ms
05-31 12:09:51.979 D/EventDiscovery-Request: (3->) GET https://api.schedjoules.com/events?latlng=52.370216,4.895168&locale=en&preceding=a1ebef892f04d227&radius=10000&u=0e61014b-bf7c-4e9d-b2be-9b4a057533b6
05-31 12:09:52.564 D/EventDiscovery-Request: (3<-) Response time: 585 ms
05-31 12:09:59.065 D/EventDiscovery-Request: (4->) GET https://api.schedjoules.com/events/27b62a8c81b00ed1/actions?u=0e61014b-bf7c-4e9d-b2be-9b4a057533b6
05-31 12:09:59.607 D/EventDiscovery-Request: (4<-) Response time: 542 ms
```